tailwind.config.js

```diff
@@ -14,8 +15,22 @@ module.exports = {
       },
       screens: {
         'max-sm': { 'max': '639px' }
+      },
+      boxShadow: {
+        white: '0 1px 3px 0 rgba(255, 255, 255, 0.1), 0 1px 2px 0 rgba(255, 255, 255, 0.06);',
+      },
+      // minHeight: Object.fromEntries([...Array(19)].map((e, i) => (i + 1) * 5).map((n) => [n, `${n}vh`]))
```

Comment:
personally, i'd prefer having hardcoded object of `minHeight`'s in `vh`. There's just something really ugly when creating a range in JS.